### DISCUSSION TITLE
have problem on new version

### DISCUSSION THREAD

**Discussion by ckokt** [swar/Swar-Chia-Plot-Manager#708] [Q&A] [answered]
i have problem on new version, why it start after 2 hours?

![swar config](https://user-images.githubusercontent.com/2990501/119179320-5ac08700-baa1-11eb-9b2f-e92914d71f93.jpg)

Equipments:
CPU: Ryzen 9-5950x - 16core
RAM: 64gb
NVME: 2tb *3
HDD: 6tb *2
Motherboard: Gigabyte X570 AORUS MASTER

this is my setting :

max_concurrent: 16
max_for_phase_1: 3
minimum_minutes_between_jobs: 5

name: p1
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: D:
temporary2_directory:
destination_directory: G:
size: 32
bitfield: true
threads: 3
buckets: 128
memory_buffer: 4000
max_concurrent: 6
max_concurrent_with_start_early: 6
initial_delay_minutes: 0
stagger_minutes: 60
max_for_phase_1: 2
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 30
temporary2_destination_sync: false
exclude_final_directory: false
skip_full_destinations: true
unix_process_priority: 10
windows_process_priority: 32
enable_cpu_affinity: false
cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31 ]

name: p2
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: E:
temporary2_directory:
destination_directory: G:
size: 32
bitfield: true
threads: 4
buckets: 128
memory_buffer: 4000
max_concurrent: 6
max_concurrent_with_start_early: 6
initial_delay_minutes: 0
stagger_minutes: 60
max_for_phase_1: 2
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 30
temporary2_destination_sync: false
exclude_final_directory: false
skip_full_destinations: true
unix_process_priority: 10
windows_process_priority: 32
enable_cpu_affinity: false
cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31 ]

name: p3
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: F:
temporary2_directory:
destination_directory: H:
size: 32
bitfield: true
threads: 5
buckets: 128
memory_buffer: 4000
max_concurrent: 6
max_concurrent_with_start_early: 6
initial_delay_minutes: 0
stagger_minutes: 60
max_for_phase_1: 2
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 30
temporary2_destination_sync: false
exclude_final_directory: false
skip_full_destinations: true
unix_process_priority: 10
windows_process_priority: 32
enable_cpu_affinity: false
cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31 ]

**Comment by Meyer85** [comment, marked as answer]
You have a stagger_minutes: 60 and max_for_phase_1: 2

Thats why

You need to change max_for_phase_1: 3 so it allows a third one to start or lower stagger times to 35 minutes so it will get out of phase one sooner

Cheers

**Reply by ckokt** [reply]
Ok, I got it. Thank you so much for your great help.
